src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -855,14 +854,36 @@ def _validate_vm_vmss_create_auth(namespace):
     elif namespace.authentication_type == 'ssh':
 
         if namespace.admin_password:
-            raise ValueError('Admin password cannot be used with SSH authentication type')
+            raise CLIError('Admin password cannot be used with SSH authentication type.')
 
         validate_ssh_key(namespace)
 
         if not namespace.ssh_dest_key_path:
             namespace.ssh_dest_key_path = \
                 '/home/{}/.ssh/authorized_keys'.format(namespace.admin_username)
 
+    elif namespace.authentication_type == 'all':
+        if namespace.os_type.lower() == 'windows':
+            raise CLIError('SSH not supported for Windows VMs. Use password authentication.')
+
+        if not namespace.admin_password:
+            _prompt_for_password(namespace)
+        _validate_admin_password(namespace.admin_password,
+                                 namespace.os_type)
+
+        validate_ssh_key(namespace)
+        if not namespace.ssh_dest_key_path:
+            namespace.ssh_dest_key_path = \
```

Comment:
Unless it is 120 characters plus, make it same line.

Reply:
I agree. I copied and pasted code from the other blocks without updating. Will make the changes.